### ISSUE TITLE
iframe error when running outside of `dsm-storybook` CLI

### ISSUE THREAD

**Issue by shilman** [InVisionApp/dsm-storybook-example-library#21]
Following up to #19, when you configure the storybook through `initDsm`, run `yarn storybook` and click on the "eject" button in the upper-right corner, the resulting iframe is empty. This breaks integrations with [Chromatic](https://www.chromaticqa.com/) and other tools.

Repro: https://github.com/shilman/dsm-test/

**Comment by mortal94** [comment]
Hey @shilman,
Thank you for reporting this bug! We've released version `0.0.105` that fixes the "eject" button new tab failing to load the iframe

**Comment by shilman** [comment]
Thanks @mortal94! Something must have been messed up on my side. Confirmed this works 👍

**Comment by shilman** [comment]
Spoke too soon. Corner case:

```bash
yarn build-storybook
npx http-server storybook-static
open http://127.0.0.1:8080/iframe.html?id=welcome--to-storybook # works
open http://127.0.0.1:8080/iframe.html #fails
```

Specifically when I go to the browser console and type `window.__STORYBOOK_STORY_STORE__` there should be stories in both the case where a query param is provided and ALSO when there is no query param.

**Comment by mortal94** [comment]
Hey @shilman! We've released version `0.0.106` that fixes the corner case when there is no query params. Thank you for notifying us!

**Comment by shilman** [comment]
Nice. DSM & Chromatic finally in harmony! Thanks for being so responsive!!! 😅
